MinimedKitUI/MinimedPumpSettingsViewController.swift

```diff
@@ -191,6 +231,14 @@ class MinimedPumpSettingsViewController: RileyLinkSettingsViewController {
         switch Section(rawValue: indexPath.section)! {
         case .info:
             break
+        case .actions:
+            switch ActionsRow(rawValue: indexPath.row)! {
+            case .suspendResume:
+                if let suspendResumeCell = sender as? SuspendResumeTableViewCell {
```

Comment:
Is there an occasion when `sender` wouldn't be an instance of SuspendResumeTableViewCell? `as!` might be appropriate here.